nucypher/blockchain/eth/agents.py

```diff
@@ -526,11 +531,11 @@ def get_staking_providers(self) -> Iterable[ChecksumAddress]:
 
     @contract_api(CONTRACT_CALL)
     def _get_active_staking_providers_raw(
-        self, start_index: int, max_results: int
+        self, start_index: int, max_results: int, duration: int
     ) -> Tuple[int, List[bytes]]:
         active_staking_providers_info = (
             self.contract.functions.getActiveStakingProviders(
-                start_index, max_results, 0  # TODO address via #3458
```

Comment:
Glad to see TODO maintenance, instead of leaving it around (even if this one is recent)!